packages/showcases/core/csf/components/actions/button.csf.js

```diff
@@ -37,6 +37,27 @@ export const argTypes = {
       options: ['small', 'medium', 'large', 'stretched'],
     },
   },
+  type: {
+    type: { name: 'string', required: false },
+    description: 'The type of the button.',
+    defaultValue: 'button',
+    control: {
+      type: 'select',
+      options: ['button', 'submit'],
+    },
+  },
```

Comment:
Why are you adding this attribute?
We already spread with `v-bind=$attrs` all the different props so I don't understand why we need to add this which is part of HTML Button specification and not Vitamin